projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -371,6 +385,7 @@ export const defaultFeatureToggles: Required<FeatureTogglesInterface> = {
   pdfInvoicesSortByInvoiceDate: false,
   storeFrontLibCardParagraphTruncated: false,
   productConfiguratorAttributeTypesV2: false,
+  productConfigurationDeltaRendering: false,
```

Comment:
Please, rename the feature toggle to `productConfiguratorDeltaRendering` because the name of our feature library is `product-configurator`.